src/backends/tensorflow/savedmodel_backend.cc

```diff
@@ -162,9 +173,19 @@ SavedModelBackend::CreateTRTISTFModel(
     const DimsList& dims =
         (io.has_reshape()) ? io.reshape().shape() : io.dims();
 
-    RETURN_IF_ERROR(CompareDimsSupported(
-        Name(), io.name(), output->shape_, dims,
-        Config().max_batch_size() > 0));
+    if (output->shape_->rank_ != 0) {
+      RETURN_IF_ERROR(CompareDimsSupported(
+          Name(), io.name(), output->shape_, dims, Config().max_batch_size() > 0));
+    }
+    else {
+      // Use output shape from model configuration if empty
```

Comment:
"The savedmodel doesn't specify a shape for the output so use the shape from the model configuration"

Reply:
Fixed comment.